log/log.go

```diff
@@ -0,0 +1,63 @@
+package log
+
+import (
+	"fmt"
+
+	"github.com/Sirupsen/logrus"
+)
+
+// Logger interface allows a logging object to be passed around as long as they
+// implement these methods without requiring a specific package.  Allows migrating
+// between logging packages along as this contract is maintained.
+type Logger interface {
+	WithField(key string, value interface{}) *logrus.Entry
+	WithFields(fields logrus.Fields) *logrus.Entry
+	WithError(err error) *logrus.Entry
```

Comment:
Return value should be `Logger` as it returns an instance of the Logger interface.